### DISCUSSION TITLE
Coming back from G7 to G6 without reinstalling xDrip?

### DISCUSSION THREAD

**Discussion by Gonzmdv** [NightscoutFoundation/xDrip#3814] [General]
After several months with G7 I'm coming back to G6. G7 is more comfortable, but I find it inaccurate and unreliable.
What settings should I change to avoid uninstalling and reinstalling xDrip?
I think I had to uninstall and reinstall xDrip the first time when I only tried G7s for a month, because xDrip was not finding the G6 after using G7s.

Thank you very much

**Comment by Navid200** [comment]
There is no need to uninstall.
You don't need to change any setting other than the transmitter ID.

**Comment by Gonzmdv** [comment]
Great! Thank you. As we need to scan a code to configure xDrip for G7, I thought there were internal changes in XDrip. If a need to change back again from G6 to G7 will it work changing only transmitter ID?

**Reply by Navid200** [reply]
The QR scan enables xDrip to use G7.  It needs to be done only once.  It does nothing else.
Therefore, after scanning, you can still use G6.
After going back to G6, to switch to G7 again, you won't need to scan the QR code again.
Even if you scan the QR code multiple times there will be no harm.

**Comment by Gonzmdv** [comment]
Thanks for the info.
